### DISCUSSION TITLE
Note off on Hydrogen 1.0.1

### DISCUSSION THREAD

**Discussion by Patsko** [hydrogen-music/hydrogen#1289] [Q&A] [answered]
Hi all, I'm looking for the "note off" feature on the pattern editor, but I can't find it on Hydrogen 1.0.1. Previously, I was using 0.9.6 which had this feature. 

This is an image from Hydrogen documentation. 
![image](https://user-images.githubusercontent.com/25756833/120712305-d9042c80-c496-11eb-9e90-47b6aa5040f0.png)

This is an image of my Hydrogen 1.0.1. There is no option to switch between note length and note off modes. Also, the right click on pattern editor doesn't add stops to the pattern.
![image](https://user-images.githubusercontent.com/25756833/120712353-e6211b80-c496-11eb-98d5-72049b0178da.png)

I haven't seen this feature on 1.1 and 1.0.2 changelogs. Do I need to go back to version 0.9.6 to have this feature?

**Comment by cme** [comment, marked as answer]
It's still there -- place or remove them with <kbd>Shift</kbd> + left-click.

(I can't remember off the top of my head what the input action in 0.9.* was to get them)

**Reply by Patsko** [reply]
That's it, thanks!

**Reply by cme** [reply]
Looking at the history, it looks like this changed between 0.9.6 and 0.9.7.

The new manual I think will make this clearer :)
